### ISSUE TITLE
Relative Json file path

### ISSUE THREAD

**Issue by SFHAJJI** [AlexTeixeira/Askmethat-Aspnet-JsonLocalizer#25]
Greate package. I'm trying to use your package in order to localize two  Json files in two different paths. I couldn't do this because in the startup.cs the resource path should be fixed.
But this is not the problem, even if i try to put the two json files under the same resource folder what i get in the result is always the content of the two Json resource files. 
I thnik that is because the basename is never user when instanciating a JsonStringLocalizer class iin the JsonStringLocalizerFactory. Create method : 
```
  public IStringLocalizer Create(string baseName, string location)
        {
            return (IStringLocalizer)new JsonStringLocalizer(_env, _memCache, _resourcesRelativePath, _localizationOptions);
        }
```
I guess it should have been something like this :
```
  public IStringLocalizer Create(string baseName, string location)
        {
            string filePath = _resourcesRelativePath+baseName
            return (IStringLocalizer)new JsonStringLocalizer(_env, _memCache, filePath, _localizationOptions);
        }
```

**Comment by AlexTeixeira** [comment]
Hi,

Thanks for using my package :).

That's true, currently baseName is not used at all, and can't be used with the current implementation.
For the location parameter, no problem it can be implemented.

From Microsoft Interface :  

```
//
        // Summary:
        //     Creates an Microsoft.Extensions.Localization.IStringLocalizer.
        //
        // Parameters:
        //   baseName:
        //     The base name of the resource to load strings from.
        //
        //   location:
        //     The location to load resources from.
        //
        // Returns:
        //     The Microsoft.Extensions.Localization.IStringLocalizer.
        IStringLocalizer Create(string baseName, string location);
```

I think the changes is not very difficult, I will try to do it soon.

For multiple JSON Path localization. Feel free to open a new issue, If you can create a pull request, otherwise I will try to do it as soon as possible.

**Comment by AlexTeixeira** [comment]
The pull request #26 will merge the changes.
Feel free to give comments about the code. If all seems ok, I will merge and deploy new version.

**Comment by AlexTeixeira** [comment]
Available on 1.1.6-preview2. 
Don't forget to give a star if you love the package 👍
